backend/repository/manager.go

```diff
@@ -321,6 +364,16 @@ func (m *Manager) UpdateAll(r *Repository, clientCfgDir, endpointCfgDir string,
 	if err := m.UpdateEndpoints(r, endpointCfgDir, req.EndpointUpdates); err != nil {
 		return errors.Wrap(err, "failed to update endpoints")
 	}
+
+	if err := m.DeleteEndpoints(r, endpointCfgDir, req.DeleteEndpoints); err != nil {
+		return errors.Wrap(err, "failed to delete endpoints")
```

Comment:
Err has been set in previous request - should be able to return `err` directly.

Reply:
Okay, I'll change this